src/main/java/digital/slovensko/autogram/model/AutogramDocument.java

```diff
@@ -29,10 +33,10 @@ public void setMasterPassword(char[] masterPassword) {
     }
 
     public char[] getSigningPassword() {
-        return openDocumentPassword.length > 0 ? openDocumentPassword : masterPassword;
+        return this.hasOpenDocumentPassword() ? openDocumentPassword : masterPassword;
```

Comment:
`this` nemá zmysel používať mimo konštruktorov, kde sa zvyknú parametre volať rovnako ako atribúty classy.

V tomto prípade to bez `this` bude robiť presne to isté.